jni/Android.mk

```diff
@@ -83,7 +83,7 @@ NNTRAINER_SRCS := $(NNTRAINER_ROOT)/nntrainer/models/neuralnet.cpp \
                   $(NNTRAINER_ROOT)/nntrainer/optimizers/optimizer_factory.cpp \
                   $(NNTRAINER_ROOT)/nntrainer/utils/util_func.cpp \
                   $(NNTRAINER_ROOT)/nntrainer/utils/parse_util.cpp \
-                  $(NNTRAINER_ROOT)/nntrainer/app_context.cpp
+									$(NNTRAINER_ROOT)/nntrainer/app_context.cpp
```

Comment:
I think this one should be a space but tab

Reply:
Fixed